### PR TITLE
step pages

### PR DESCRIPTION
**Pull request by stormcloud266**

### CODE REVIEW

File: web/src/pages/FormArchetypePage/FormArchetypePage.tsx [hunk 1]
```diff
@@ -210,12 +195,14 @@ function StepForm({
...
-  const { fields: stepPageFields, append: appendStepPageField } = useFieldArray(
-    {
-      control,
-      name: `${stepsArrayName}.${index}.stepPage`,
-    }
-  )
+  const {
+    fields: stepPageFields,
+    append: appendStepPageField,
+    remove: removeStepPageField,
```
Comment (stormcloud266) [inline web/src/pages/FormArchetypePage/FormArchetypePage.tsx:201]: We're adding it here, just renaming it removeStepPageField so it doesn't conflict with the other remove function

File: web/src/pages/FormArchetypePage/FormArchetypePage.tsx [hunk 2]
```diff
@@ -16,8 +16,6 @@ import {
   CreateStepTokenIdRangeInput,
   CreateStepOriumApiInput,
   CreatePuzzleInput,
-  // left off here on 12/28/2023
-  CreateStepPageInput,
```
Comment (Richard-Burd) [inline web/src/pages/FormArchetypePage/FormArchetypePage.tsx:20]: how are we grabbing this now?

Comment (stormcloud266) [reply web/src/pages/FormArchetypePage/FormArchetypePage.tsx:20]: This is the thing Bloom added to the `CreateStepInput` sdl this morning. So now we don't need to grab it directly, it just comes with the `CreateStepInput` by default

**Review by Richard-Burd — APPROVED** [review]
looks good to me.  I would like to know where you're grabbing the info for `CreateStepPageInput` (line 20 of form)

**Review by illepic — APPROVED** [review]

**Pull request merged**